app/services/articles/enrich_image_attributes.rb

```diff
@@ -37,22 +34,14 @@ def self.call(article)
 
         next if image.blank?
 
-        attribute_width, attribute_height = image_width_height(img)
-        img["width"] = attribute_width
-        img["height"] = attribute_height
-        img["data-animated"] = true if FastImage.animated?(image, timeout: TIMEOUT, raise_on_failure: false)
+        property = FastImage.new(image, timeout: 10)
```

Comment:
I find the singular `property` a bit confusing here, how about `properties` or `img_properties`?

Reply:
Yup I can change that